tests/integration/components/frost-bunsen-form/renderers/multi-select/options/ember-data-view-query-test.js

```diff
@@ -0,0 +1,1291 @@
+import {expect} from 'chai'
```

Comment:
This file is so huge because I started out with a `select` renderer test and then just switched it to `multi-select` and removed bits that weren't appropriate for a multi-select. It's probably more tests than we need for this fix, but they should probably be here anyway.